src/algorithms/sauvola.jl

```diff
@@ -94,11 +94,9 @@ end
 
 Sauvola(; window_size::Int = 7, bias::Real = 0.2) = Sauvola(window_size, bias)
 
-function binarize(algorithm::Sauvola, img::AbstractArray{T,2}) where T <: Colorant
-    binarize(algorithm, Gray.(img))
-end
-
```

Comment:
This line is redundant according to https://codecov.io/gh/zygmuntszpak/ImageBinarization.jl/src/master/src/algorithms/sauvola.jl